api/v1_coin_insights.go

```diff
@@ -12,8 +12,7 @@ type ArtistCoinInsights struct {
 }
 
 type MembersStatsRow struct {
-	Members                 int     `json:"members"`
-	Members24hChangePercent float64 `json:"membersChange24hPercent"`
+	Members int `json:"members"`
```

Comment:
could we remove this too @faridsalau / @DejayJD ? Or are we using it somewhere? Might be nice to just have this be a birdeye proxy